mappings/net/minecraft/loot/function/ApplyBonusLootFunction.mapping

```diff
@@ -1,28 +1,28 @@
 CLASS net/minecraft/class_94 net/minecraft/loot/function/ApplyBonusLootFunction
 	FIELD field_1009 formula Lnet/minecraft/class_94$class_96;
 	FIELD field_1010 FACTORIES Ljava/util/Map;
-	FIELD field_1011 enchantment Lnet/minecraft/class_1887;
+	FIELD field_1011 type Lnet/minecraft/class_1887;
```

Comment:
This field is called `enchantment` in json format. https://minecraft.gamepedia.com/Loot_table#Functions